semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/Scalacp.scala

```diff
@@ -496,10 +496,21 @@ object Scalacp {
           sys.error(s"unsupported kind $skind for symbol $sym")
       }
     }
+    def semanticdbDecls: SemanticdbDecls = {
+      val decls = sym.children.filter(decl => decl.isUseful && !decl.isTypeParam)
+      SemanticdbDecls(decls.toList)
+    }
   }
 
-  private implicit class ScopeOps(decls: Seq[Symbol]) {
-    def useful: Seq[Symbol] = decls.filter(decl => decl.isUseful && !decl.isTypeParam)
+  case class SemanticdbDecls(syms: List[Symbol]) {
+    lazy val ssyms: List[String] = {
+      val sbuf = List.newBuilder[String]
```

Comment:
why not `syms.map(ssymbol)`?

Reply:
This implementation gets an upgrade in a subsequent commit. It looks dumb when introduced, but that allows for an easier to read diff later on.